Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py

```diff
@@ -30,8 +40,7 @@ def __init__(self, aws_default_region, aws_role_arn, aws_role_session_name, aws_
         self.aws_role_session_duration = aws_role_session_duration
         self.aws_role_policy = aws_role_policy
         self.aws_access_key_id = aws_access_key_id
-        self.aws_secret_access_key = aws_secret_access_key
-        self.aws_session_token = aws_session_token
+        self.aws_secret_access_key, self.aws_session_token = extract_session_from_access(aws_secret_access_key, aws_session_token)
```

Comment:
I think we need to fix `aws_session` function. It looks like the code won't go into the part to log in with access_key + session_token. See: https://github.com/demisto/content/blob/f24dbfef8a47c6cc2f6665dfb89dd3438af410be/Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py#L137

Reply:
I fixed it